src/lib/bootloader/grub2_widgets.rb

```diff
@@ -423,6 +423,42 @@ def validate
     end
   end
 
+  # Represents switcher for NVRAM update
+  class UpdateNvramWidget < CWM::CheckBox
+    include Grub2Widget
+
+    def initialize
+      textdomain "bootloader"
+    end
+
+    def label
+      _("Update &NVRAM Entry")
+    end
+
+    def help
+      res = _("<p><b>Update NVRAM</b> will add nvram entry for the bootloader\n" \
+          "in the firmware.\n" \
+          "This is usually desirable unless you want to preserve specific settings\n" \
+          "or need to work around firmware issues.\n" \
+          "\n")
+      res += "</p>"
```

Comment:
three notes to this help text:

1. having `<p>` in localized text and `</p>` not is strange and can confuse translators
2. it is richtext so `\n` is not needed at all
3. `Update NVRAM` does not match `Update NVRAM Entry`. bold text should mention widget name.

Reply:
The texts commonly include \n on end of line.